### ISSUE TITLE
PHP shebang echoed during commit message validation

### ISSUE THREAD

**Issue by danepowell** [acquia/blt#1657]
BLT 8.9.0-beta1

When I make a modification to a BLT-based project and commit the changes, I see the following echoed in the output to the terminal:

> ...
Linting YAML files...
Skipped pre-commit target hook. No hook is defined.
Your local code has passed git pre-commit validation.
#!/usr/bin/env php
Validating commit message syntax...
...

Obviously that PHP shebang shouldn't be there. It's happening because of this require_once: https://github.com/acquia/blt/blob/8.x/scripts/git-hooks/commit-msg#L16

Basically, you are including a shell-like PHP script from within a PHP environment, so PHP echos the shebang as if it were HTML. See https://stackoverflow.com/a/10731074/2433782

I'm not sure the best way to fix it... maybe separate the core php functionality and shell wrappers in php and shell-like scripts, respectively?

**Comment by grasmash** [comment]
Yeah I noticed this.
